### PR TITLE
Add baidu cloud vector search in vectorstore and fix some unit test in vectorstores

### PR DESCRIPTION
**Pull request by wemysschen**
**Description:** 
Add baidu cloud vector search in vectorstore

### CODE REVIEW

**Comment by wemysschen** [comment]
@baskaryan Hi, could you please review code again?

**Comment by wemysschen** [comment]
@baskaryan   Could you please review code again?

File: libs/langchain/langchain/vectorstores/baiducloud_vector_search.py [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+from langchain.schema.embeddings import Embeddings
+from langchain.schema.vectorstore import VectorStore
+
+if TYPE_CHECKING:
+    from elasticsearch import Elasticsearch
+
+logger = logging.getLogger(__name__)
+
+
+class BESVerctorStore(VectorStore):
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/baiducloud_vector_search.py:25]: BESVectorStore

**Comment by baskaryan** [comment]
looks great, thank you @wemysschen!

**Pull request merged**